spec/tests/ConvivaAnalytics.spec.ts

```diff
@@ -8,28 +9,510 @@ jest.mock('@convivainc/conviva-js-coresdk', () => {
 });
 jest.mock('../../src/ts/Html5Logging');
 
-describe('content metadata spec', () => {
+describe(ConvivaAnalytics, () => {
```

Comment:
GH tracks this file with lots of changes, but there are not so many new tests. I'll comment under them so it's easier to navigate.